app/scenes/Collection.js

```diff
@@ -188,26 +190,31 @@ class CollectionScene extends React.Component<Props> {
             {collection.isEmpty ? (
               <Centered column>
                 <HelpText>
-                  <strong>{collection.name}</strong> doesn’t contain any
-                  documents yet.
-                  <br />
-                  Get started by creating a new one!
+                  {t(
```

Comment:
Another spot where JSX snuck into the string

Reply:
Again, same as above.

Reply:
gonna fix it